pkg/virt-operator/resource/apply/pdb_test.go

```diff
@@ -93,10 +92,6 @@ var _ = Describe("Apply PDBs", func() {
 		kv.Status.TargetKubeVirtVersion = Version
 		kv.Status.TargetDeploymentID = Id
 
-		mockPodDisruptionBudgetCacheStore = &MockStore{}
-		err := stores.PodDisruptionBudgetCache.Add(mockPodDisruptionBudgetCacheStore)
```

Comment:
Could you please add a few words about why is this code redundant?

Reply:
Sure. There are couple if issues here:
1. `stores.PodDisruptionBudgetCache` is already assigned a cache store, there is no need to re-assign it again.
2. Even if we want to re-assign, that should be like this: `stores.PodDisruptionBudgetCache = &MockStore{}` and by adding the cache store object as an element to the cache store itself.
3. There was no assertion on the returned error value.